### DISCUSSION TITLE
[灌水] 新款邦布！有谁入手了没？

### DISCUSSION THREAD

**Discussion by wugan-bot156** [share121/inter-knot#583] [其他]
![屏幕截图 2024-08-10 115211](https://github.com/user-attachments/assets/d431308d-4802-4282-a9b2-aacbc9c16fff)
有谁入手玛瑟尔HP-601c了没?就是海报上这个新款邦布！看介绍很带劲啊！最新的以骸感知警报器，1秒极速导入⌜萝卜⌟地图，超长待机空洞导航，4个自定义模组扩展槽，十六款换装…有没有用过的人给个评测?这一眼看上去全是优点啊！

**Comment by DiamondDiavolo** [comment]
未下单，期待收货

**Comment by LadySunBringer** [comment]
感谢支持我们玛瑟尔集团的产品🥰

**Comment by GUILAISHIXV** [comment]
未付款，期待发货

**Comment by KEARUM** [comment]
这只是预售广告吧，应该还没人拿到手？楼主还是先观望观望吧

**Comment by FrostPhoenix79** [comment]
插个眼，有没有用过的绳匠分享一下使用体验。如果不错的话，孩子砸锅卖铁也要攒出来。

**Comment by heibaiya-dev** [comment]
拿到啦！！！！！！！
抽到测试资格了
保密协议不能发图片
实测大型萝卜0.89秒导入
小型萝卜0.12秒导入
pro版和普通版都不同
pro版可以做到0.4秒导入超大型萝卜
待机更长
抗摔抗侵蚀，pro版有8个自定义插槽，测试人员和我透露max有16个
简直跨时代
下空洞表现超级好
缺点：贵 贵 贵

**Reply by heibaiya-dev** [reply]
邦布开发平台还提供多种api端口，有钱值得入手

**Comment by Alanhhhhhhh** [comment]
请问到外环包邮吗？

**Comment by a0906390287** [comment]
已下單期待與馬瑟爾小姐一起起飛

**Comment by TianyiLuo0721** [comment]
正在攒钱买😉

**Comment by Palaluo** [comment]
o<img src="https://bbs-static.miyoushe.com/static/2024/10/24/0296f44f6aa17344294e4ccca4829b5a_2147609872749895683.png" alt="悠闲01" width="75" >
